hsctl

```diff
@@ -227,37 +229,42 @@ rebuild_nodb_hs() {
 
 loaddb_hs() {
     echo "*** ${1^^} ***"
-    echo "*** load clean pg.development.sql database from the running hydroshare container ***"
-    CID=$(docker-compose ps | grep hydroshare | cut -d ' ' -f 1)
+    echo "*** load ${HS_DATABASE} database file from the running hydroshare container ***"
     echo "*** drop existing database ***"
-    docker exec $CID dropdb -U postgres -h postgis postgres
+    docker exec hydroshare dropdb -U postgres -h postgis postgres
     echo "*** create new database ***"
-    docker exec $CID createdb -U postgres -h postgis postgres --encoding UNICODE --template=template0
-    echo "*** create POSTGIS extension ***"
-    docker exec $CID psql -U postgres -h postgis -w -c 'create extension postgis;'
+    docker exec hydroshare createdb -U postgres -h postgis postgres --encoding UNICODE --template=template0
+    echo "*** INFO: SET client_min_messages = ERROR on PostgreSQL ***"
+    docker exec hydroshare psql -U postgres -h postgis -w -c 'SET client_min_messages = ERROR;'
+    echo "*** INFO: CREATE EXTENSION postgis on PostgreSQL ***"
+    docker exec hydroshare psql -U postgres -h postgis -w -c 'CREATE EXTENSION postgis;'
+    echo "*** INFO: CREATE EXTENSION hstore on PostgreSQL ***"
+    docker exec hydroshare psql -U postgres -h postgis -w -c 'CREATE EXTENSION hstore;'
```

Comment:
During my testing of hstore extension I had to run the following command in order for the test database to also have access to the hstore extension (note the template option):
   
     psql -U postgres -h postgis -d template1 -w -c 'create extension hstore;'